generated/user_existence/user_existence.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I think we also need an example of the return value you get if the user is NOT in enroll, right? 

And we probably want to treat those as two different use cases (so perhaps instead of `user_existence`, we want a `user_exists_in_enroll_IVL`, `user_exists_in_enroll_SHOP`, and `user_not_in_enroll` directories